### PR TITLE
Finished the responsive design of my website

### PR DESCRIPTION
**Pull request by Medini-Rajendra**
## In thie review, I have done the following 
 - I have finished making my website with responsive design. 
 - Both mobile and desktop versions work 
 - Checked for linter errors. 
 - Added all design elements.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -784,3 +784,808 @@ textarea {
...
+.desk_projects-container1,
+.desk_projects-container2,
+.desk_projects-container3,
+.desk_languages-container,
+.contactform_end {
+  display: none;
+}
+
+/* starting with responsive web design */
+@media screen and (min-width: 600px) {
```
Comment (arslanbisharat) [inline style.css]: You have used ` @media queries` for the Desktop breakpoint in the CSS file, Great Job! 👏 

- [ ] Make sure the value for **min-width** property should be in range of `992px - 1100px`

File: index.html [hunk 2]
```diff
@@ -20,12 +20,152 @@
...
+                    <li class="contact-form-name">
+                        <input type="text" maxlength="30" class="nooutline" id="name_grid" name="user_name" placeholder="Your name">
+                    </li>
+                    <li class="contact-form-email"><input type="email" id="mail_grid" class="nooutline" name="user_email" placeholder="Your email"></li>
+                    <li class="contact-form-message"><textarea maxlength="500" id="msg_grid" class="nooutline" name="user_message" placeholder="Enter your message here..."></textarea></li>
+                </ul>
+            </form>
+                <img class="getstarted_image" src="image-geometry_7.svg" alt="Gridimage">
+                <div class="footer_rect"></div>
+                <div class="text17_footer">Get my Resume</div>
```
Comment (arslanbisharat) [inline index.html]: Get my Resume looks just like the design. :clap:

- [x] Make sure this is a button.,

File: index.html [hunk 3]
```diff
@@ -20,12 +20,152 @@
...
+                </ul>
+            </div>
+            <img class="newimg_grid33" src="Group_84.svg" alt="gridimage">
+            <img class="newimg1_grid33" src="Shape_grid33.svg" alt="gridimage">
+            <a name="abouttext">
+                <div class="abouttext">About</div>
+            </a>
+            <div class="aboutpara">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</div>
+            <div class="rectangle14_grid33">
+                <div class="getmycv">Get my CV</div>
```
Comment (arslanbisharat) [inline index.html]: This is great you followed the design here and it looks amazing. 

- [ ]  Make sure it's a button.

File: index.html [hunk 4]
```diff
@@ -20,12 +20,152 @@
             <div class="defineimage"></div>
             <div class="defineimage1"></div>
         </header>
+        <div class="horizontal_line">
+            <a href="#text1" class="wrap ite1">Hello</a>
+            <a href="#text3" class="wrap ite2">Portfolio</a>
+            <a href="#abouttext" class="wrap ite4">About</a>
+            <a href="#text10" class="wrap ite5">Contact</a>
+        </div>
```
Comment (arslanbisharat) [inline index.html:28]: You have beautifully managed to align this section into the center. Amazing job. 👏 

- [x] Make sure the link blue text is removed and it follow the design color.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi @mach512, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[ ] [Optional] It would be great if you can add all the images in a single directory. This will help you to present your repo profesionally. 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited numbers of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 4 above)
Comment (Medini-Rajendra) [reply index.html:28]: I have fixed the issue and changed the color.

File: index.html (hunk 2 above)
Comment (Medini-Rajendra) [reply index.html]: I have made this as button.

File: style.css (hunk 1 above)
Comment (Medini-Rajendra) [reply style.css]: I have changed this media screen min-width.

File: index.html (hunk 3 above)
Comment (Medini-Rajendra) [reply index.html]: I have changed it to the button as per your suggestion.

**Review by Shaher-11 — APPROVED** [review]
#### Hello @Medini-Rajendra,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle::muscle:

### Optional Suggestions💡
_The following point is optional to implement as the previous reviewer did not mention it however it is highly recommended!_:smile::smile:

- Kindly center the bottom section as shown in the original design below
#### Original Project
right
#### Your Project
wrong



_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**